cmake/GinkgoConfig.cmake.in

```diff
@@ -174,6 +170,7 @@ endif()
 # For details, see https://gitlab.kitware.com/cmake/cmake/issues/18614
 if((NOT GINKGO_BUILD_SHARED_LIBS) AND GINKGO_BUILD_CUDA)
     enable_language(CUDA)
+    find_package(CUDAToolkit REQUIRED)
     find_package(NVTX REQUIRED)
```

Comment:
should we also use `CUDA::nvToolsExt` when cmake < 3.25 and `CUDA::nvtx3` when cmake >= 3.25?

Reply:
I think as long as we need to support both versions (which require different includes), it's easier to use the existing FindNVTX module.